style1.css

```diff
@@ -0,0 +1,243 @@
+@import url('https://fonts.googleapis.com/css2?family=Spinnaker&display=swap');
+/* navbar style */
+.navbar{
+    display: flex;
+    position: sticky;
+    width: 100%;
+    padding: 20px 5px;
+    z-index: 1;
+    background-color: white;
+    box-shadow: 0 3.5px 3.5px -2px rgba(94, 93, 93, 0.2);
+}
+.nav__items-container{
+    flex-direction: row !important;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.logo h2{
+    display: flex;
+}
+.logo a{
+    font-size:25px ;
+    font-family: "palatino linotype", sens-serif;
+    color: #474646;
+    text-decoration: none;
+}
+.nav__tabs{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.nav__item{
+    display: inline-block;
+    padding: 0px 15px;
+}
+.nav__item a{
+    font-size:15px ;
+    font-family: spinnaker, sens-serif;
+    color: #6e6e6e;
+    list-style-type: none;
+    text-decoration: none;
+}
+.login-icon a,
+.cart-icon a{
+    color: #5386e4;
+    font-family: "palatino linotype", sens-serif;
+    font-size: 20px;
+    text-decoration: none;
+}
+.bars-icon{
+    display: flex;
+}
+.bars-icon i{
+    font-size: 20px;
+}
+/* banner style */
+.banner{
+    display: flex;
+    justify-content: center;
+    position: relative;
+    width: 100%;
+}
+.ban__landing-container{
+    display: flex;
+    flex-direction: row !important;
+    justify-content: center;
+    align-items: flex-start;
+    padding: 0px !important;
+}
+.ban__landing-img1{
+    position: absolute;
+    height: auto;
+    width: 100%;
+}
+.ban__landing-img2{
+    position: absolute;
+    height: auto;
+    width: 100%;
+    display: none;
+}
+.ban__content{
+    display: block;
+    margin: 0px 14px;
+    word-spacing: 5px;
+    position: absolute;
+    color: white;
+    text-align: center;
+    line-height: 35px;
+    margin-top: 55px;
+} 
+.ban__content h3,.ban__content h1,.ban__content h3 a{
+    display: block;
+}
+.ban__content h3{
+font-size: 16px;
+font-weight: 100;
+font-family: spinnaker, sens-serif;
+}
+.ban__content h3 a{
+    color: #35548f;
+}
+.ban__content h1{
+    font-size: 90px;
+    font-family: "palatino linotype", sens-serif;
+}
+/* media query */
+@media only screen and (max-width:600px){
+    .nav__items-container .nav__tabs ul{
+        display: none;
+    }
+    .ban__landing-img2{
+        display:block;
+    }
+    .ban__landing-img1{
+        display: none;
+    }
+    .logo a{
+        font-size: 30px;
+    }
+    .ban__content h3,.ban__content h3 a{
+        font-size: 12px;
+    }
+    .ban__content h1{
+        font-size: 37px;
+    }
+    .ban__content{
+        line-height: 18px;
+        letter-spacing: 1px ;
+        margin-top: 25px;
+    }
+}
+@media only screen and (min-width:601px){
```

Comment:
why change breakpoint?

Reply:
because when screen size is exactly 600px both of the bar menu and the tabs will become visible at this size